ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcHttpServiceTest.java

```diff
@@ -1699,8 +1699,20 @@ private void assertTransactionResultMatchesTransaction(
       assertThat(result.getValue("to")).isNull();
     }
     assertThat(Wei.fromHexString(result.getString("value"))).isEqualTo(transaction.getValue());
-    assertThat(Wei.fromHexString(result.getString("gasPrice")))
-        .isEqualTo(transaction.getGasPrice());
+    Optional.ofNullable(result.getString("gasPrice"))
+        .ifPresent(
+            maxFeePerGas ->
+                assertThat(Wei.fromHexString(maxFeePerGas)).isEqualTo(transaction.getGasPrice()));
+    Optional.ofNullable(result.getString("maxFeePerGas"))
+        .ifPresent(
+            maxPriorityFeePerGas ->
+                assertThat(Wei.fromHexString(maxPriorityFeePerGas))
+                    .isEqualTo(transaction.getMaxFeePerGas().get()));
+    Optional.ofNullable(result.getString("maxPriorityFeePerGas"))
+        .ifPresent(
+            gasPriceString ->
+                assertThat(Wei.fromHexString(gasPriceString))
+                    .isEqualTo(transaction.getMaxPriorityFeePerGas().get()));
```

Comment:
these lambda parameter names are a bit jumbled.  Test should pass, but gasPrice -> maxFeePerGas, maxFeePerGas -> maxPriorityFeePerGas, maxPriorityFeePerGas -> gasPriceString

🤯

Reply:
wow, how did I manage to do that haha